apps/compute-cf.yml.j2

```
@@ -179,7 +179,7 @@ Resources:
             Resource: !Sub "arn:aws:s3:::${ContentBucket}/*"
           - Effect: Allow
             Action: sns:Publish
-            Resource: !Sub "arn:aws:sns:${AWS::Region}:${AWS::AccountId}:*"
+            Resource: !Sub "arn:aws:sns:*"
```

No need for a `!Sub` here anymore
```suggestion
            Resource: arn:aws:sns:*
```